packages/terra-functional-testing/lib/docker/docker-compose.yml

```
@@ -0,0 +1,29 @@
+version: '3.7'
+
+services:
+  hub:
+   image: selenium/hub:3.141.59-zirconium
+   ports:
+     - "4444:4444"
+
+  chrome:
+    image: selenium/node-chrome:3.141.59-zirconium
```

do we want the latest?
```suggestion
    image: selenium/node-chrome:3.141.59-20200525
```
https://github.com/SeleniumHQ/docker-selenium/releases/tag/3.141.59-20200525

---

I'll update the image, I had thought the date images were beta releases. It looks like they switched and adopted the date suffix:

https://github.com/SeleniumHQ/docker-selenium/wiki/Tagging-Convention